src/components/hmi_message_handler/test/hmi_message_handler_impl_test.cc

```diff
@@ -189,6 +189,38 @@ TEST_F(HMIMessageHandlerImplTest, SendMessageToHMI_Success) {
   EXPECT_TRUE(waiter.WaitFor(1, 100));
 }
 
+TEST(WebsocketSessionTest, SendMessage_Success) {
+  ::testing::FLAGS_gtest_death_test_style = "threadsafe";
+
+  auto send_message = []() {
+    auto message =
+        "{\"id\" : 1,\"jsonrpc\" : \"2.0\",\"method\" : "
+        "\"BasicCommunication.GetSystemInfo\"}";
+
+    boost::asio::io_context ioc{1};
+    boost::asio::ip::tcp::acceptor acceptor{
+        ioc, {boost::asio::ip::make_address("127.0.0.1"), 8087}};
+    boost::asio::ip::tcp::socket socket{ioc};
+
+    Json::Reader reader;
+    Json::Value json_value;
+    if (!reader.parse(message, json_value, false)) {
+      return;
+    }
+
+    auto session =
+        new hmi_message_handler::WebsocketSession(std::move(socket), nullptr);
+    session->sendJsonMessage(json_value);
+    std::this_thread::sleep_for(std::chrono::milliseconds(100));
```

Comment:
@SNiukalov is it enough time? Think about slow CI

Reply:
@AKalinich-Luxoft 
I think it is enough, I looked others tests relative  SendMessage, they have same timeout and good working on CI.